Competition2017/.classpath

```diff
@@ -12,5 +12,9 @@
 	<classpathentry kind="lib" path="/SeriouslyCommonLib/lib/WPILib-sources.jar"/>
 	<classpathentry kind="lib" path="/SeriouslyCommonLib/lib/WPILib.jar" sourcepath="/SeriouslyCommonLib/lib/WPILib-sources.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="var" path="wpilib" sourcepath="wpilib.sources"/>
```

Comment:
eclipse auto generates these, we should avoid checking them in, maybe adding a checkstyle rule would help guard things